Ensembl_to_NCBI_AM_061324.py

```diff
@@ -0,0 +1,71 @@
+#Ensembl to NCBI Accession Codes for homo sapiens. 
+#This is just my first attempt at this kind of project and am only aiming to print out the ID's for NCBI. 
+#Example: ENSG00000012048 
+import requests
+import re
+from Bio import Entrez
+import urllib.parse
+import json
+import xml.etree.ElementTree as ET
+
+Entrez.email = input("In compliance with NCBI policy, please input your email:") 
+e_id = input("Please type in a valid Ensembl ID:").strip() 
+#For future scripts, will try to have this be a list or data frame column to loop through 
+# but let's just focus on getting this to work with one for now. - 061324
+
+def database_search(gene_id):
+    
+    url = f"https://rest.ensembl.org/lookup/id/{gene_id}?content-type=application/json"
+    summary_url = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi" 
```

Comment:
You can place these into a constants.py file. This is more from a readability perspective/code organization perspective. What you did here is totally fine.